### PR TITLE
EDSC-2812: Adds test for each file in serverless

### PR DESCRIPTION
**Pull request by abbottry**
# Overview

### What is the feature?

Ensuring our lambdas are well tested.

### What is the Solution?

Adds a test file for every file in the serverless directory.

### What areas of the application does this impact?

The entire serverless directory.

# Testing

### Reproduction steps

1. `npm run tests` will now show a test file for ever file/function in serverless.

### Attachments

Please include relevant screenshots or files that would be helpful in reviewing and verifying this change.

# Checklist

- [x] I have added automated tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

**Comment by abbottry** [comment]
#1227 needs to be complete and merged before this can be merged.

File: static/src/js/components/CustomToggle/MoreActionsToggle.js [hunk 1]
```diff
@@ -1,44 +1,39 @@
...
-        onClick={onClick}
-        title="More actions"
-        icon="ellipsis-v"
-      />
-    )
-  }
-}
-
-ToggleMoreActions.defaultProps = {
+// eslint-disable-next-line react/display-name
```
Comment (macrouch) [inline static/src/js/components/CustomToggle/MoreActionsToggle.js:9]: Why is this needed here and in GranuleResultsDataLinksButton.js but not in CustomToggle.js ?

**Review by trevorlang — APPROVED** [review]
Looks good

**Review by macrouch — APPROVED** [review]
